hazelcast/src/test/java/com/hazelcast/internal/cluster/impl/PromoteLiteMemberTest.java

```diff
@@ -323,6 +326,86 @@ public void promotion_shouldFail_whenMasterIsSuspected_duringPromotion() throws
         }
     }
 
+    @Test
+    public void lite_member_promotion_data_loss() throws InterruptedException {
```

Comment:
can we give a better name to test? Seeing the state tested against and expected behavior in test method name would be helpful for the future readers.